capability/index.md

```diff
@@ -1,9 +1,12 @@
 ---
 layout: default
 title: Capability
-nav_order: 2
+nav_order: 3
 parent: RRAP M&DS
-has_children: 
+has_children: true
 ---
 
-# RRAP M&DS Capability 
\ No newline at end of file
+# Capability 
+This section details specific use cases of the M&DS capability. 
+
+The M&DS suites provides knowledge and advice across multiple study areas to inform and support strategic and tactical intervention decisions on The Reef.
```

Comment:
Suggest "The Reef" -> "the Great Barrier Reef"?

Reply:
Noted & updated